src/MovieCard.js

```diff
@@ -2,6 +2,7 @@ import React from "react";
 import "./MovieCard.css";
 
 const MovieCard = ({
+  onClick,
```

Comment:
Be more specific with this function/prop name. You might have a bunch of click handlers in your application, and you'll want to name the function/prop more specific to what the click handler is doing.